File: sample/Python/kabusapi_unregisterall.py
import urllib.request
import json
import pprint

url = 'http://localhost:18080/kabusapi/unregister/all'
req = urllib.request.Request(url, method='PUT')
req.add_header('Content-Type', 'application/json')
req.add_header('X-API-KEY', 'e629f7e6073b40488d0d134dae4e60ac')
# req.add_header('Content-Length', 0)

try:
    with urllib.request.urlopen(req) as res:
        print(res.status, res.reason)
        for header in res.getheaders():
            print(header)
        print()
        content = json.loads(res.read())
        pprint.pprint(content)
except urllib.error.HTTPError as e:
    print(e)
    content = json.loads(e.read())
    pprint.pprint(content)
except Exception as e:
    print(e)

File: sample/C#/CSharp/Kabusapi_Unregisterall.cs
using System;
using Newtonsoft.Json;
using System.Net.Http;
using System.Text;

namespace CSharp_sample
{
    class Kabusapi_Unregisterall
    {
        static void Main(string[] args)
        {
            var url = "http://localhost:18080/kabusapi/unregister/all";
            string token = GenerateToken.GetToken();
            try
            {
                var client = new HttpClient();
                var request = new HttpRequestMessage(HttpMethod.Put, url);
                request.Headers.Add("ContentType", "application/json");
                request.Headers.Add("X-API-KEY", token);
                HttpResponseMessage response = client.SendAsync(request).Result;
                Console.WriteLine("{0} \n {1}", JsonConvert.DeserializeObject(response.Content.ReadAsStringAsync().Result), response.Headers);

            }
            catch (HttpRequestException e)
            {
                Console.WriteLine("{0} {1}", e, e.Message);
                Console.ReadKey();
            }
            catch (Exception ex)
            {
                Console.WriteLine("{0} {1}", ex, ex.Message);
                Console.ReadKey();
            }
            Console.ReadKey();
        }
    }
}
